content/docs/started/k8s/overview.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "Kubeflow on Existing Kubernetes Clusters"
```

Comment:
Change the title so that it doesn't repeat the section name. Suggestion:

"Deploying Kubeflow on Existing Clusters"

Reply:
Updated. Please have a look